### PR TITLE
mongo-sanitize: a more accurate export

### PR DESCRIPTION
**Pull request by penumbra1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by penumbra1** [comment]
I never got the old typings to work with es6 imports.
Based on [this SO answer](https://stackoverflow.com/a/47406161), there is no need for `export as namespace mongoSanitize;` as mongo-sanitize is not a UMD module but a single CommonJS statement: 
```js
module.exports = function(v) {
  // ...
}
```
The [README](https://github.com/DefinitelyTyped/DefinitelyTyped/#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default) says:
> Do not change the type definition if it is accurate. For an NPM package, export = is accurate if node -p 'require("foo")' works to import a module

In this case it does.

I'm not sure if anyone ever got this working with the old namespace, and removing it may have been a breaking change. Please correct me if I'm wrong, I'm a noob:)

File: types/mongo-sanitize/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // Type definitions for mongo-sanitize 1.0
 // Project: https://github.com/vkarpov15/mongo-sanitize
-// Definitions by: Cedric Cazin <https://github.com/CedricCazin>
+// Definitions by: Cedric Cazin <https://github.com/CedricCazin>, Olga Isakova <https://github.com/penumbra1>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.3
```
Comment (sandersn) [inline types/mongo-sanitize/index.d.ts]: what makes 3.3 required instead of 2.3?

**Review by sandersn — CHANGES_REQUESTED** [review]
Looks good except for the requirement of TS 3.3. Seems like 2.3 is still good enough.

**Comment by penumbra1** [comment]
Yes, sorry I misinterpreted the comment. 2.3 is still fine,

File: types/mongo-sanitize/index.d.ts (hunk 1 above)
Comment (penumbra1) [reply types/mongo-sanitize/index.d.ts]: Yes, sorry I misinterpreted the comment (as the latest tested version as opposed to required). 2.3 is fine.

Comment (penumbra1) [reply types/mongo-sanitize/index.d.ts]: However, --esModuleInterop was introduced in [2.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html), and this is when namespace imports started throwing "no call signature".  Does this mean 2.7 a requirement? 
The CI check fails for the same reason and will pass with 2.7. I've rolled back to 2.3 for now, but please let me know if I should change the version to 2.7.

**Review by sandersn — APPROVED** [review]

**Pull request merged**